src/application.c

```diff
@@ -396,7 +400,9 @@ bool app_init(struct config* cfg, const char** sources, size_t num)
 
     // setup window position and size
     if (ctx.window.width != SIZE_FULLSCREEN) {
```

Comment:
I explicitly made the macro conditional around the swaywm code and not the if statement, since its not specific to sway.

Reply:
What I meant was that 'if' is not needed if nothing depends on it.
```
#ifdef ....
if (ctx.window.width != SIZE_FULLSCREEN) {
```
instead of
```
if (ctx.window.width != SIZE_FULLSCREEN) {
#ifdef ....
```

Reply:
> What I meant was that 'if' is not needed if nothing depends on it.

I know. I'm saying that the code is not explicitly associated with the Sway feature. Only wanted to point that out, since i presume this causes compiler warnings.

Reply:
This `if` is _explicitly_ associated with the Sway feature.